plugins/DuskVerb/tools/tuner/t60_sweep.py

```
@@ -0,0 +1,125 @@
+        "--param", "Bus Mode=On",
+        "--param", f"Decay Time={decay}",
+        "--param", f"Bass Multiply={bass}",
+        "--param", f"Treble Multiply={treble}",
+        "--param", f"Mid Multiply={mid}",
+        "--param", f"Hi Cut Shelf={hicut_shelf}",
+        "--param", f"Low Crossover={low_xover}",
+        "--prerun-seconds", "5",
+        "--output-dir", str(out_dir),
+    ]
+    try:
+        return subprocess.run(cmd, capture_output=True, text=True, timeout=120).returncode == 0
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)